zuul.d/project-templates.yaml

```diff
@@ -597,13 +592,9 @@
     name: ansible-collections-vyos-vyos
     check:
       jobs: &ansible-collections-vyos-vyos-jobs
-        - ansible-test-network-integration-vyos-local-python27-stable210
```

Comment:
I think we should keep stable branches

Reply:
Did you mean this to be on 2.7, or the 3.6 runs below?

Reply:
yah, we still want to support python27 and local on stable branches. We've only dropped py27 on devel right now

Reply:
So... are we not dropping any local tests on vyos then?

Reply:
yah, I think we need to keep all on vyos, but can drop them every place else.